plugins/net_plugin/net_plugin.cpp

```diff
@@ -3378,38 +3417,21 @@ namespace eosio {
    // called from application thread
    void net_plugin_impl::on_accepted_block(const block_state_ptr& bs) {
       update_chain_info();
-      dispatcher->strand.post( [this, bs]() {
+      dispatcher->strand.post( [bs]() {
```

Comment:
small detail: maybe capture `bs` by reference.

Reply:
`bs` is a `const&` we have to capture it by value since the lambda is executed async.

Reply:
got it, thanks!